doc/md/guides/quickstart.mdx

```diff
@@ -114,9 +134,14 @@ Run the rest of the commands in this guide from the root of the repository.
 The Bank of Holos platform is organized as a collection of software components.
 Each component represents a piece of software provided by an upstream vendor,
 for example ArgoCD, or software developed in-house.  Components are also used to
-glue together, or integrate, other components into the platform.
+glue together, or integrate, other components into the platform. ArgoCD is a good
+first example for a platform because:
+
+  - It's a component that's easily deployed via an upstream Helm chart.
```

Comment:
Don't call it a component yet, we're mixing in Application resources to other components.  That is to say, we're configuring other components.